### PR TITLE
Add `c:experience` fluid tag with javadoc for standard conversion rate

### PR DESCRIPTION
**Pull request by TelepathicGrunt**
By default in past, the conversion rate was 20mb to 1 experience. But this feels a bit odd. Should we define it to 1mb to 1 experience so it is clearer and more intuitive? Also better handles when someone has say 17mb left.

I'll do Fabric PR later with rest of the missing fluid tags to them that Neo has. Their conversion rate can be different since they operate on a different base unit than Neo. This tag is just to say what the fluid is. But within the loader itself, the javadoc should say what the conversion rate that loader's community uses.

### CODE REVIEW

File: src/main/java/net/neoforged/neoforge/common/Tags.java [hunk 1]
```diff
@@ -790,6 +790,11 @@ public static class Fluids {
          * (Standard unit for honey bottle is 250mb per bottle)
          */
         public static final TagKey<Fluid> HONEY = tag("honey");
+        /**
+         * Holds all fluids related to experience.<br></br>
```
Comment (Technici4n) [inline src/main/java/net/neoforged/neoforge/common/Tags.java]:
```suggestion
         * Holds all fluids related to experience.
```
What the hell :D

**Comment by Technici4n** [comment]
I think the standard unit of 20 mb is good.

**Comment by HenryLoenwind** [comment]
That 20:1 ratio gives you sensible fluid amounts. But it is a PITA to handle conversions without a buffer tank to hold those 1-19mB remainders...

**Comment by Flanks255** [comment]
I also think the existing rate of 20:1 is sufficient.
The reasoning of 1mb being better for odd amounts could be applied to any fluid in the game, water comes in 1000mb buckets, but things can use partial amounts, or it could be split in pipes just as often as experience, same with any other fluid conversion, ingots, nuggets, redstone.
In a continuous system, IE a mob farm or anything where you actually would have pipework, there will be more experience coming  to top off the partial amounts if it got split for some reason.

**Review by Shadows-of-Fire — APPROVED** [review]

**Review by Technici4n — APPROVED** [review]

**Pull request merged**